services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -33,6 +35,7 @@
  * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
  */
 @Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "title", name = "uniqueTitle"))
```

Comment:
Or I could simply assume that DB constraint error is for title unique index as there's nothing else.